packages/cli/src/util/client.ts

```diff
@@ -1,21 +1,19 @@
 import { MedplumClient, MedplumClientOptions } from '@medplum/core';
 import { FileSystemStorage } from '../storage';
 
-export async function createMedplumClient(options: MedplumClientOptions): Promise<MedplumClient> {
-  const baseUrl = options.baseUrl ?? process.env['MEDPLUM_BASE_URL'] ?? 'https://api.medplum.com/';
-  const fhirUrlPath = options.fhirUrlPath ?? process.env['MEDPLUM_FHIR_URL_PATH'] ?? '';
-  const accessToken = options.accessToken ?? process.env['MEDPLUM_CLIENT_ACCESS_TOKEN'] ?? '';
-  const tokenUrl = options.tokenUrl ?? process.env['MEDPLUM_TOKEN_URL'] ?? '';
-  const authorizeUrl = options.authorizeUrl ?? process.env['MEDPLUM_AUTHORIZE_URL'] ?? '';
-  const fetchApi = options.fetch ?? fetch;
+export async function createMedplumClient(options: MedplumClientOptions, profileName?: string): Promise<MedplumClient> {
+  const profile = profileName ?? 'default';
+  const storage = new FileSystemStorage(profile);
+  const { baseUrl, fhirUrlPath, accessToken, tokenUrl, authorizeUrl, fetchApi } = getClientValues(options, storage);
 
   const medplumClient = new MedplumClient({
     fetch: fetchApi,
     baseUrl,
     tokenUrl,
     fhirUrlPath,
     authorizeUrl,
-    storage: new FileSystemStorage(),
+    storage,
+    // TODO: add authType to Medplum
```

Comment:
@jamestouri  - I don't think this is actually a TODO.  I think `authType` will be used at `login` time.

`medplum login` - default behavior, prompt user to login, use `medplum.startLogin`

`medplum login --auth-type basic --client-id XYZ --client-secret 123` - use `medplum.setBasicAuth()`

`medplum login --auth-type client-credentials --client-id XYZ --client-secret 123` - use `medplum.startClientLogin()`

`medplum login --auth-type jwt-bearer --client-id XYZ --client-secret 123` - use `medplum.startJwtBearerLogin()`